### ISSUE TITLE
Aliased import with leading underscore produces private usage error

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#912]
**Describe the bug**
When importing a public variable using an 'as' clause, assigning it to a name with a leading underscore and accessing it, Pyright reports that the variable is private and used outside of the module in which it is declared.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That public variables bound to a locally private name would not be analysed as being originally private.  It also might make more sense for the error to be produced on import rather than each time the variable is used.

**Screenshots or Code**
```py
# pyright: strict

from builtins import str as _foo

_foo

```

**VS Code extension or command-line**
Extension, 1.1.59

**Comment by erictraut** [comment]
Good catch. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is addressed in 1.1.60, which I just published. It will also be in the next version of Pylance.
